compiler/test-resources/repl/i21657

```diff
@@ -0,0 +1,2 @@
+scala> :sh
```

Comment:
```suggestion
scala>:sh
```

Not sure why this is true, but something makes it so we can't have that nice first space char